lib/ohloh_scm/adapters/git_svn/pull.rb

```diff
@@ -0,0 +1,83 @@
+module OhlohScm::Adapters
+  class GitSvnAdapter < AbstractAdapter
+    def pull(source_scm, &block)
+      @source_scm = source_scm
+      convert_to_git(&block)
+    end
+
+    def branch_name
+      'master'
+    end
+
+    private
+
+    def convert_to_git(&block)
+      yield(1, 2) if block_given?
```

Comment:
Change this to `yield (0, 1)` for consistency with other adapters. This is what we do in GitAdapter#pull.